### ISSUE TITLE
On Sony Projectors Blanking then Unblank doesn't un-mute. 

### ISSUE THREAD

**Issue by xuther** [byuoitav/av-api#141]
We probably have to have some logic in our unblank command evaluators that generates an unmute command when we unblank for Sony Projectors - as it appears that the rooms don't unmute after an unblank command is sent.

**Comment by c-Monster** [comment]
Is this still an issue?

**Comment by xuther** [comment]
Yup. This'll be a change in the unblank command evaluator for the Sony Projector rooms so it generates two commands - an unmute and an unblank - each time the user requests and unblank.

**Comment by xuther** [comment]
closed in error. Still needs a fix
